.github/workflows/docker-publish.yaml

```diff
@@ -0,0 +1,60 @@
+name: Docker
+
+on:
+  push:
+    tags:
+      - v*
+
+jobs:
+  # Push container image to GitHub Packages and Docker Hub.
+  # See also https://docs.docker.com/docker-hub/builds/
+  deploy:
+    runs-on: ubuntu-latest
+    if: github.event_name == 'push'
+
+    strategy:
+      matrix:
+        images:
+          - wildbook-ia
+          - wbia-base wbia-dependencies wbia-provision wbia
+
+    steps:
+      - uses: actions/checkout@v2
+
+      # Build images
+      - name: Build images
+        run: bash devops/build.sh ${{ matrix.images }}
+
+      # Log into container registries
+      - name: Log into Docker Hub
+        run: echo "${{ secrets.WBIA_WILDMEBOT_DOCKER_HUB_TOKEN }}" | docker login -u wildmebot --password-stdin
+
+      - name: Log into GitHub Packages
+        run: echo "${{ secrets.GITHUB_TOKEN }}" | docker login docker.pkg.github.com -u ${{ github.actor }} --password-stdin
+
+      # Push containers out to container registries
+      - name: Push to GitHub Packages
+        if: ${{ github.event_name == 'push' && startsWith(github.event.ref, 'refs/tags/v') }}
```

Comment:
This file looks almost the same as nightly.  Why don't we just add this to nightly?